index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <script src="https://kit.fontawesome.com/4d799c321f.js"></script>
+    <title>Intuit Accounts - Sign Up</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <section>
+        <div class="navbar">
+
+            <div class="links">
+                <p> Already have an Joaze account? </p>
+                <a href="#">
+                    I forgot my User ID or Password Sign In
+                </a>
+            </div>
+
+            <div class="navBtn">
+                <button><i class="fas fa-lock"></i>Sign In</button>
+            </div>
+
+        </div>
+
+        <header class="mainHead">
+
+            <div class="title">
+                <h1>
+                    <a href="#">
+                        J O A Z E
+                    </a>
+                </h1>
+            </div>
+
+            <div class="c">
```

Comment:
This section should have the original should have the icon text like the original page.

#### Make sure
 - [ ] It shows the Mint logo and text (logo on the left, text on the right).

 - [ ] It shows the Quickbooks logo and text (logo on the left, text on the right).

 - [ ] It shows the Turbotax logo and text (logo on the left, text on the right).

Reply:
We realised that in the original page, these where images and we have  now included them